### ISSUE TITLE
Why Hotspot is creating by default

### ISSUE THREAD

**Issue by GokulG07** [3zuli/esp_internet_alarm#2]
Hi Thanks for this Little Project....I'm have a issue that after uploading the code the Node Mcu esp8266 is creating a hotspot......I have earlier tested a esp8266 hotspot project, but i have erased everything from the esp.....Now at this point i don't know what should be done....and also i thing this 4 year old project need some feature update's......
Thank you

**Comment by tromano32** [comment]
I was looking at this project also but not have had chance to try it... I do know that your esp might have remnants of old code in the WIFI section.. In Arduino, under the tool menu.... make sure you set your "Erase Flash" option to "All Flash Contents" and be sure to specify the SSID and PASSWORD in the code it should connect right away.

**Comment by GokulG07** [comment]
yes, it was the old code, that problem was solved, now I have the connectivity issue as it is not connecting to My wifi sometimes and I have to manually restart the esp to connect again, its an everyday problem don't know what the cause!

**Comment by tromano32** [comment]
I think some esp8266 are better than others When i comes to stability.... maybe look into adding the watchdog library and have it reboot every few days ... that’s my plan

**Comment by GokulG07** [comment]
yeah, that will work...could be hours rather than days, better than alarming NO Internet whole day. anyways share your code's here it can help all...Thanks

**Comment by 3zuli** [comment]
Sorry for not replying earlier. I haven't worked on this project for 4 years and the code is really outdated. Given how much the Arduino ESP ecosystem has evolved, the code should probably be completely rewritten at this point. Also I originally had this running on an ESP-01 module, but nowadays I would recommend to use a D1 Mini board or something similar, as it's much more convenient. Then you should also use a different pin than GPIO02.
And actually I also remember having some reliability issues. The ESP-01 would frequently crash on some watchdog timeout, and for some reason it was refusing to connect to certain WiFi networks. I think this could have been fixed since then in newer versions of the Arduino ESP8266 plugin...

**Comment by tromano32** [comment]
> Sorry for not replying earlier. I haven't worked on this project for 4 years and the code is really outdated. Given how much the Arduino ESP ecosystem has evolved, the code should probably be completely rewritten at this point. Also I originally had this running on an ESP-01 module, but nowadays I would recommend to use a D1 Mini board or something similar, as it's much more convenient. Then you should also use a different pin than GPIO02.
> And actually I also remember having some reliability issues. The ESP-01 would frequently crash on some watchdog timeout, and for some reason it was refusing to connect to certain WiFi networks. I think this could have been fixed since then in newer versions of the Arduino ESP8266 plugin...

Thanks... I plan on using your code as just an example .. most of it will get rewritten and I will utilize WS2812 NEOPIXEL LED's on a D1 Mini...
